plugins/janus_sip.c

```diff
@@ -1713,7 +1723,9 @@ void janus_sip_incoming_rtp(janus_plugin_session *handle, int video, char *buf,
 			JANUS_LOG(LOG_ERR, "No session associated with this handle...\n");
 			return;
 		}
-		if(session->status != janus_sip_call_status_incall)
+		if(session->status != janus_sip_call_status_incall &&
+			session->status != janus_sip_call_status_incall_reinviting &&
+			session->status != janus_sip_call_status_incall_reinvited)
```

Comment:
I see there are a lot of checks now that include all the three states, since `janus_sip_call_status_incall` is not enough anymore. Maybe it would be simpler and more readable to add a static helper method that does that? e.g., `janus_sip_is_incall(session)`?

Reply:
I did consider doing it and but thought that incorporating all three states into `janus_sip_is_incall` might sound misleading  or confusing because method `janus_sip_is_incall` would sound like a shortcut for a single state `janus_sip_call_status_incall`. So wasn't sure it would be a good idea. Maybe, we can name the new method `janus_sip_call_is_established`?